class2.1/task3.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include <fstream>
+#include <map>
+#include <string>
+//I create history.txt file for saving data.
+using namespace std;
+
+map<string, int> userHistory;
+
+void loadHistory() {
+    ifstream file("history.txt");
```

Comment:
Magic numbers and strings like this(e.g. hardcoded constants, hardcoded file paths or commands) should be moved to anonymous namespace as `constepxr auto defaultUserHistoryFile = "hiistory.txt";`
Also review other code - "delete" and "bread" commands also should be some named constants - it will allow to change them easiily if needed.